jnotebook-core/src/main/java/tech/catheu/jnotebook/NotebookRenderer.java

```diff
@@ -89,7 +90,11 @@ public void render(final Main.RenderConfiguration config) {
         html = optimizeHtml(html);
       }
 
-      final File outputFile = FileUtils.getFile(config.outputPath);
+      var outputPath = config.outputPath;
+      if(outputPath==null) {
```

Comment:
nit for me: introduce 

LangUtils.java
```
public static Optional<T> optional(T o) {
  return Optional.ofNullable(o);
}
```
and use it.

Reply:
not sure what you mean if outputPath is nullable the above is the simplest code?

Reply:
I was meaning: 
```
final String outputPath = optional(config.outputPath)
  .orElse(Files.getNameWithoutExtension(config.inputPath) + ".html");
```

but it's for me later no worries.